src/lexer/token.rs

```diff
@@ -43,7 +43,7 @@ pub enum TokenKind {
   ReturnKeyword,
 
   // Non-zoglin
-  CommandBegin,
+  CommandBegin(bool),
```

Comment:
Do we need to store this `bool` in the token? It doesn't seem like it's used anywhere

Reply:
It's used in `tokenise` to determine whether the command literal uses a backtick or is just a keyword command.

Reply:
It probably can and should be refactored, but that was the easiest implementation without changing how commands were set to be lexed.